### ISSUE TITLE
Links are not clickable in Editor with edit.off()

### ISSUE THREAD

**Issue by Driesigner** [froala/wysiwyg-editor#4286]
##### Expected behavior.
When I set FroalaEditor in read-only mode with the edit.off() method, the links in the content should be clickable.

##### Actual behavior.
When I set FroalaEditor in read-only mode with the edit.off() method, the links in the content are not clickable. Right-clicking and selecting 'Open Link in new tab' works.

##### Steps to reproduce the problem.
* Add a link-tag in a Froala-Editor
* Configure FroalaEditor to edit-off().
* Click the link in the disabled FroalaEditor

##### Editor version.
Seen in: react-froala-wysiwyg v3.2.7
I expect it's not related to the react-implementation.

##### OS.
MacOS

##### Browser.
Tested in Google Chrome and Firefox

**Comment by st-abarker** [comment]
I can confirm that this issue is not specific to the react implementation. We use angular-froala with froala-editor and we are seeing the same behavior.

**Comment by NitrusAphalion** [comment]
Facing the same issue. Tested on latest 4.0.3 and issue still exists. 

Fiddle: https://jsfiddle.net/voqcfsyd/

**Comment by Driesigner** [comment]
I was in contact with Froala support about this issue. It will be fixed in release 4.0.5
In the meantime, this fix works for me (I'm using a loop because I have multiple instances of froala on one page):

```typescript
events: {
    initialized() {
      if (readOnly) {
        const frViews = document.getElementsByClassName('fr-view');
        if (frViews) {
          for (let i = 0; i < frViews.length; i++) {
            frViews[i].setAttribute('contenteditable', 'false');
          }
        }
      }
    },
  },
```

**Comment by Driesigner** [comment]
> I was in contact with Froala support about this issue. It will be fixed in release 4.0.5
> In the meantime, this fix works for me (I'm using a loop because I have multiple instances of froala on one page):
> 
> ```ts
> events: {
>     initialized() {
>       if (readOnly) {
>         const frViews = document.getElementsByClassName('fr-view');
>         if (frViews) {
>           for (let i = 0; i < frViews.length; i++) {
>             frViews[i].setAttribute('contenteditable', 'false');
>           }
>         }
>       }
>     },
>   },
> ```

**Comment by Driesigner** [comment]
Fix has been delayed to 4.0.6

**Comment by st-abarker** [comment]
We just received an update from support. As they related to me:

> Closed by the dev-team with the comment below:
> this is a change in existing behavior and could cause regressions for other users. Suggest the users to implement a preview option (https://jsfiddle.net/froala/tewf1kxn/) to meet the requirement
